src/storage/mod.rs

```diff
@@ -1073,6 +1199,72 @@ impl Storage {
             .flatten()
     }
 
+    pub fn async_raw_batch_scan(
+        &self,
+        ctx: Context,
+        start_keys: Vec<Vec<u8>>,
+        start_key_limit: usize,
```

Comment:
What about `each_limit`?

Reply:
sure, let me change it.

Reply:
Changed to each_limit, kvproto was updated as well. It's gonna take an hour for CI to finish I guess.

Reply:
CI succeeded this time.